sequel/benchmarks/bm_sequel_destroy.rb

```diff
@@ -0,0 +1,26 @@
+require 'bundler/setup'
+require 'sequel'
+require_relative 'support/benchmark_rails'
+
+DB = Sequel.connect(ENV.fetch('DATABASE_URL'))
+
+DB.create_table(:users) do
+  primary_key :id
+  column :name, "varchar(255)"
+  column :email, "varchar(255)"
+  column :created_at, "timestamp", null: true
+  column :updated_at, "timestamp", null: true
+end
+
+class User < Sequel::Model; end
+
+attributes = {
+  name: "Lorem ipsum dolor sit amet, consectetur adipiscing elit.",
+  email: "foobar@email.com"
+}
+
+Benchmark.sequel("sequel/#{db_adapter}_destroy", time: 5) do
+  # we need to create the record in order to delete it
+  user = User.create(attributes)
```

Comment:
For sequel, we need [raise_on_save_failure](http://sequel.jeremyevans.net/rdoc/classes/Sequel/Model/ClassMethods.html). Otherwise, saving a record may fail giving us an inaccurate benchmark